frontend/src/container/Sidebar.jsx

```diff
@@ -42,9 +49,28 @@ export class Sidebar extends React.Component {
     this.handleClickNewWorkspace()
   }
 
-  handleOpenWorkspaceInSidebar = data => {
-    this.props.dispatch(setWorkspaceListIsOpenInSidebar(data.openInSidebarId, true))
-    if (data.openInSidebarId && document.getElementById(data.openInSidebarId)) document.getElementById(data.openInSidebarId).scrollIntoView()
+  handleTlmMemberCreated = tlmFieldObject => {
+    const { props } = this
+
+    const tlmUser = tlmFieldObject.user
+    const tlmAuthor = tlmFieldObject.author
+    const tlmWorkspace = tlmFieldObject.workspace
+    const loggedUserId = props.user.userId
+
+    if (loggedUserId === tlmUser.user_id) {
+      props.dispatch(addWorkspaceList([tlmWorkspace]))
+      props.dispatch(addWorkspaceMember(tlmUser, tlmWorkspace.workspace_id, tlmFieldObject.member))
+
+      // INFO - CH - 2020-06-25 - if logged used is author of the TLM and the new role is for him, it means the logged
+      // user created a new workspace
+      if (loggedUserId === tlmAuthor.user_id) {
```

Comment:
This mean that when a workspace is added by a user, each window where the user is connected to Tracim will be redirected to the dashboard of the new workspace ?

Reply:
indeed, it's an issue.
I'm going to implement the identification through the client_token